release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously the `create agent-config-template` command printed a line with `INFO` only but no details about whether the command was successful and where the template file was written to. Now, if the command is successful, the command will print `INFO Created Agent Config Template in <path> directory`. (link:https://issues.redhat.com/browse/OCPBUGS-13408[*OCPBUGS-13408*])
+
+* Previously, when a user specified the 'vendor' hint in the `agent-config.yaml` file, the value was being checked against the wrong field so that the hint would not match. With this update, the use of the `vendor` hint correctly selects a disk. (link:https://issues.redhat.com/browse/OCPBUGS-13356[*OCPBUGS-13356*])
+
+* For clusters that run on {rh-openstack} that you upgrade to 4.14 and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in link:https://access.redhat.com/solutions/7013893[OpenShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
+
+* Previously, setting the `metadataService.authentication` field to `Required` when installing a cluster on AWS did not configure the bootstrap VM to use IMDSv2 authentication. This could result in installations failing if you configured your AWS account to block IMDSv1 authentication. With this update, the `metadataService.authentication` field correctly configures the bootstrap VM to use IMDSv2 authentication when set to `Required`. (link:https://issues.redhat.com/browse/OCPBUGS-12964[*OCPBUGS-12964*])
+
+* Previously, if you configured user-defined outbound routing when installing a private Azure cluster, the cluster was incorrectly deployed with the default public load balancer. This behavior occurred when using the installer-provisioned infrastructure to install the cluster. With this update, the installation program no longer creates the public load balancer when user-defined routing is configured. (link:https://issues.redhat.com/browse/OCPBUGS-9404[*OCPBUGS-9404*])
+
+* Previously, the vSphere Terraform `vsphere_virtual_machine` resource did not include the `firmware` parameter. This issue caused the firmware of the virtual machine (VM) to be set to `bios` by default instead of `efi`. Now, the resource includes the `firmware` parameter and sets `efi` as the default value for the parameter, so that the VM runs the Extensible Firmware Interface (EFI) instead of the basic input/output system (BIOS) interface. (https://issues.redhat.com/browse/OCPBUGS-9378[OCPBUGS-9378*)
```

Comment:
```suggestion
* Previously, the vSphere Terraform `vsphere_virtual_machine` resource did not include the `firmware` parameter. This issue caused the firmware of the VM to be set to `bios` by default instead of `efi`. Now, the resource includes the `firmware` parameter and sets `efi` as the default value for the parameter, so that the VM runs the Extensible Firmware Interface (EFI) instead of the basic input/output system (BIOS) interface. (https://issues.redhat.com/browse/OCPBUGS-9378[OCPBUGS-9378*)
```